lib/health_checks/rails.rb

```diff
@@ -16,17 +19,39 @@ def rails
 
   private
 
-  class RedisCheck < OkComputer::Check
-    def check
-      sidekiq_response = ::Sidekiq.redis(&:ping)
+  module Logger
+    module_function
 
-      if sidekiq_response == 'PONG'
-        mark_message 'Connected to Redis'
-        return
-      end
+    def error(error)
+      Rails.logger.error "Error: #{error}"
+    end
+  end
 
+  class CustomMongoidCheck < OkComputer::MongoidCheck
+    def initialize(db)
+      @db_name = db[:name]
+      client_name = "#{@db_name}_rails_health_check"
+      self.session = MongoidCustomClient.create(client_name, db[:hosts])
+    end
+
+    def check
+      mongodb_name
```

Comment:
What's this function? Is its coming from `OkComputer::MongoidCheck`? If so - document what it does, it's far from obvious. Better yet, if this is the only reason you inherit from OkComputer modules - copy-paste the implementation here and possibly improve it. It makes the code easier to read and allows future improvements

Reply:
yeah, good point. I will document it for now. Seems better to use the gem functionality as it is supposedly improved/checked for bugs once in a while